awscli/customizations/wizard/core.py

```diff
@@ -24,38 +24,42 @@ def __init__(self, step_handlers):
         # These will hold the values we store from
         # running through the plan steps.
         self._parameters = {}
+        self._current_step_index = -1
```

Comment:
Seems a little strange to track these at the object level considering they're unique to a specific `run` call.